packages/vite-plugin-react-router/README.md

```diff
@@ -38,6 +37,115 @@ export default defineConfig({
 })
 ```
 
+Your app is ready to [deploy to Netlify](https://docs.netlify.com/deploy/create-deploys/).
+
+### Deploying to Edge Functions
+
+By default, this plugin deploys your React Router app to
+[Netlify Functions](https://docs.netlify.com/functions/overview/) (Node.js runtime). You can optionally deploy to
+[Netlify Edge Functions](https://docs.netlify.com/edge-functions/overview/) (Deno runtime) instead.
+
+First, toggle the `edge` option:
+
+```typescript
+export default defineConfig({
+  plugins: [
+    reactRouter(),
+    tsconfigPaths(),
+    netlifyReactRouter({ edge: true }), // <- deploy to Edge Functions
+    netlify(),
+  ],
+})
+```
+
+Second, you **must** provide an `app/entry.server.tsx` (or `.jsx`) file that uses web-standard APIs compatible with the
+Deno runtime. Create a file with the following content:
+
+> [!IMPORTANT]
+>
+> This file uses `renderToReadableStream` (Web Streams API) instead of `renderToPipeableStream` (Node.js API), which is
+> required for the Deno runtime. You may customize your server entry file, but see below for important edge runtime
+> constraints.
+
+```tsx
```

Comment:
identical to the Cloudflare one in the RR7 codebase... https://github.com/remix-run/react-router/blob/cb9a090316003988ff367bb2f2d1ef5bd03bd3af/integration/helpers/vite-plugin-cloudflare-template/app/entry.server.tsx